nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -1494,12 +1494,40 @@ public void interactionWithContracts() {
         .doTest();
   }
 
+  @Test
+  public void testForGuavaRawTypeBuildIssue() {
```

Comment:
Rather than pulling in Guava here, let's just test for the core issue of static method calls of the form `X.foo()` where `X` is there explicitly.  You should be able to write a standalone test case that exposes the previous crash, without importing Guava or adding the `makeHelperWithGuava()` method

Reply:
I agree. I have changed the test case to reproduce this scenario without using importing guava and ImmutableSet. Let me know if this works.